docs/user_guide/admin_commands.rst

```diff
@@ -53,10 +51,15 @@ commands shown as examples of how they may be run with a description.
     ,``pwd clientname``,Print the name of workspace root directory
     sys_info,``sys_info server``,Get system information
     ,``sys_info client *clientname*``,Get system information. Individual clients can be shutdown by specifying *clientname*.
+    remove_client,``remove_client clientname``,Issue command for server to release client before the 10 minute timeout to allow client to rejoin after manual restart
```

Comment:
Do we have explanation somewhere to clarify this command?

As what this command does is remove the current "clientname" entry in server, but if receiving heartbeats / connections from the newly started client with same name it will still show up.